float64 angle
---
bool success
string message